### ISSUE TITLE
Single image inference example?

### ISSUE THREAD

**Issue by antithing** [Daniil-Osokin/gccpm-look-into-person-cvpr19.pytorch#14]
Hi, and thank you for making this code available. 
What do i need to do to run inference (using the pre-trained model) on my own random image as cv::Mat?

Do I also need to provide a bounding box?

Thank you.

**Comment by Daniil-Osokin** [comment]
Hi! It needs general steps for this group of methods: to find the pose, person need to be centered inside the image, and only the pose of this one person will be found. There is [OpenVINO demo](https://github.com/opencv/open_model_zoo/tree/master/demos/python_demos/single_human_pose_estimation_demo) that automates step: runs person detector and for each found person runs pose estimation, however it requires to use models in internal format. If you want to do it by hands, I suggest to do the following:
* Make a crop of person from an image, so person will be centered inside this crop.
* [Normalize](https://github.com/Daniil-Osokin/gccpm-look-into-person-cvpr19.pytorch/blob/master/datasets/lip.py#L46) the crop and make a tensor to feed the network.
* Use routines from `val.py` to [extract](https://github.com/Daniil-Osokin/gccpm-look-into-person-cvpr19.pytorch/blob/master/val.py#L73-L88) keypoints and [visualize](https://github.com/Daniil-Osokin/gccpm-look-into-person-cvpr19.pytorch/blob/master/val.py#L98-L115) them.

**Comment by Daniil-Osokin** [comment]
Hope, it is clear now.
